lib/chef-dk/command/generator_commands/cookbook.rb

```diff
@@ -77,6 +85,7 @@ def initialize(params)
         def run
           read_and_validate_params
           if params_valid?
```

Comment:
You cant see it in this diff but below you would only print the valid options if !params_valid?